x-pack/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/top_values/top_values.tsx

```diff
@@ -43,32 +43,32 @@ export const TopValues: FC<Props> = ({ stats, fieldFormat, barColor }) => {
     isTopValuesSampled,
   } = stats;
   const progressBarMax = isTopValuesSampled === true ? topValuesSampleSize : count;
-
   return (
     <div data-test-subj="mlFieldDataCardTopValues">
-      {topValues.map((value: any) => (
-        <EuiFlexGroup gutterSize="xs" alignItems="center" key={value.key}>
-          <EuiFlexItem grow={false} style={{ width: 100 }} className="eui-textTruncate">
-            <EuiToolTip content={kibanaFieldFormat(value.key, fieldFormat)} position="right">
-              <EuiText size="xs" textAlign="right" color="subdued">
-                {kibanaFieldFormat(value.key, fieldFormat)}
+      {Array.isArray(topValues) &&
+        topValues.map((value: any) => (
+          <EuiFlexGroup gutterSize="xs" alignItems="center" key={value.key}>
+            <EuiFlexItem grow={false} style={{ width: 100 }} className="eui-textTruncate">
```

Comment:
With the full width content in the expanded row, you can probably increase the width of the label, to give more space for text like URIs:

![image](https://user-images.githubusercontent.com/7405507/102096972-37701c80-3e1d-11eb-8209-9deeda797bb9.png)